Mage/src/main/java/mage/game/events/LifeLostBatchEvent.java

```diff
@@ -0,0 +1,69 @@
+package mage.game.events;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * @author jimga150
+ */
+public class LifeLostBatchEvent extends GameEvent implements BatchGameEvent<LifeLostEvent> {
+
+    private final Set<LifeLostEvent> events = new HashSet<>();
+
+    public LifeLostBatchEvent(LifeLostEvent event) {
+        super(EventType.LOST_LIFE_BATCH, null, null, null);
+        addEvent(event);
+    }
+
+    @Override
+    public Set<LifeLostEvent> getEvents() {
+        return events;
+    }
+
+    @Override
+    public Set<UUID> getTargets() {
+        return events.stream()
+                .map(GameEvent::getTargetId)
+                .filter(Objects::nonNull)
+                .collect(Collectors.toSet());
+    }
+
+    @Override
+    public int getAmount() {
+        return events
+                .stream()
+                .mapToInt(GameEvent::getAmount)
+                .sum();
+    }
+
+    public int getAmount(UUID targetID) {
+        return events
+                .stream()
+                .filter(ev -> ev.getTargetId().equals(targetID))
+                .mapToInt(GameEvent::getAmount)
+                .sum();
+    }
+
+    public boolean isCombatDamage() {
+        return events.stream().anyMatch(LifeLostEvent::isCombatDamage);
```

Comment:
Can be wrong usage. Related card must use events list or use method like getLifeLoseByCombatDamage(isCombatDamage::Boolaean) > 0, not isCombatDamage.